examples/stable-diffusion/README.md

```diff
@@ -897,6 +897,66 @@ python image_to_video_generation.py \
     --width=512 \
     --height=512
 ```
+# i2vgen-xl
+i2vgen-xl is high quality Image-to-Video synthesis via cascaded diffusion models. Please refer to  [Huggingface i2vgen-xl doc](https://huggingface.co/ali-vilab/i2vgen-xl).
+
+## Single Image Prompt
+
+Here is how to generate video with one image and text prompt:
+
+```bash
+PT_HPU_MAX_COMPOUND_OP_SIZE=1 \
+python image_to_video_generation.py \
+    --model_name_or_path "ali-vilab/i2vgen-xl" \
```

Comment:
let's add this model (or a smaller one) to the examples testing suite

Reply:
Ok. I will add.

Reply:
Done.